pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -60,55 +59,57 @@ Some simplifications require more than one step. For example:
 ]
 
 An expression that is not a @tech{value} can always be partitioned
-into two parts: a @deftech{redex}, which is the part that changed in a
+into two parts: a @deftech{redex} (``reducible expression''),
+which is the part that can change in a
 single-step simplification (highlighted), and the
 @deftech{continuation}, which is the evaluation
-context surrounding an expression. In @racket[(- 4 (+ 1 1))], the redex is @racket[(+ 1 1)], and
-the continuation is @racket[(- 4 @#,hole)], where @hole takes the
-place of the redex. That is, the continuation says how to ``continue''
+context surrounding the redex. In @racket[(- 4 (+ 1 1))], the redex is @racket[(+ 1 1)], and
+the continuation is @racket[(- 4 @#,hole)], where @hole indicates the
+position of the redex. That is, the continuation says how to ``continue''
 after the @tech{redex} is reduced to a @tech{value}.
 
-Before some things can be evaluated, some sub-expressions must be
-evaluated; for example, in the application @racket[(- 4 (+ 1 1))], the
+Before some expressions can be evaluated, their sub-expressions must be
+evaluated. For example, in the application @racket[(- 4 (+ 1 1))], the
```

Comment:
I’d be careful here—the original wording is admittedly vague, but perhaps intentionally so. When evaluating an expression like `(lambda (x) (+ 1 x))`, the sub-expression `(+ 1 x)` is *not* evaluated, and indeed, it cannot be. Therefore, not all sub-expressions are necessarily evaluated when an expression is evaluated. This is also true for forms like `if`, which evidently must not have a standard evaluation order. Perhaps there is some way to reword this paragraph to avoid the vague phrase “some sub-expressions”, but I think this is too aggressive.

Reply:
+1, I like this new rewording.